examples/tutorials/05_train_sklearn_models.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p><code>sklearn</code></p><p>Adding highlighting for names of modules, methods, etc might be a good idea</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>